modules/authentication/src/handlers/local.ts

```diff
@@ -512,7 +515,7 @@ export class LocalHandlers implements IAuthenticationStrategy {
       '+hashedPassword',
     );
     if (isNil(user)) throw new GrpcError(status.NOT_FOUND, 'User not found');
-    if (isNil(user.hashedPassword))
+    if (isNil(user?.hashedPassword))
```

Comment:
This is redundant. We already verify user exists in the above line. If an assertion is required, use `!`.